README copy.md

```diff
@@ -0,0 +1,42 @@
+# Portfolio mobile set up
+This project was done with html and css.
+
+## Built With
+
+- html
+- css
+
+
+## Getting Started
+
+To get a local copy up and run git clone git clone https://github.com/stevenmukama/portfoliomobile.git and follow these simple steps
+
+### Open files in visual studio code
+
+### Run the index.html file on any browser
+
+
+👤 **Author1**
+
+- GitHub: [@stevenmukama](https://github.com/stevenmukama)
+- Twitter: [@mukama_steven](https://twitter.com/mukama_steven)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Great job adding the MIT file, but you forgot to customize it, so I suggest adding your name and customizing it 💯

![image](https://user-images.githubusercontent.com/45231276/202827969-ca406f94-2af2-4c78-8a93-c448eb6269e9.png)